features/graphql/query.feature

```diff
@@ -49,6 +49,16 @@ Feature: GraphQL query support
             }
           }
         }
+        nestedCollection {
+          name
+        }
```

Comment:
you can't fetch a collection without using edges in graphql usually or can you?

Reply:
you can control it for all collections via the `paginationEnabled` flag. With the flag enabled you get edges / nodes structure. With the flag disabled you get the result in a `collection` node